ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/clientlibs/editor/js/accordion.js

```diff
@@ -0,0 +1,275 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2019 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+(function($, Coral) {
+    "use strict";
+
+    var selectors = {
+        dialogContent: ".cmp-accordion__editor",
+        edit: {
+            childrenEditor: "[data-cmp-is='childrenEditor']",
+            singleExpansion: "[data-cmp-accordion-v1-dialog-edit-hook='singleExpansion']",
+            expandedItems: "[data-cmp-accordion-v1-dialog-edit-hook='expandedItems']",
+            expandedSelect: "[data-cmp-accordion-v1-dialog-edit-hook='expandedSelect']",
```

Comment:
add dependencies of site, whatever we can reuse as is.

Reply:
removed this file and added sites library as dependency as there is no changes in both libraries.